velox/connectors/hive/SplitReader.h

```diff
@@ -110,6 +111,12 @@ class SplitReader {
   std::string toString() const;
 
  protected:
+  void createReader();
```

Comment:
```
nit: s/createReader/createFileReader/
```

Can you comment these functions? Thanks!

Reply:
@xiaoxmeng The `createReader()` is to create the `Reader` object as specified in the interface velox/dwio/common/Reader.h. Similarly, `createRowReader()` is to create the `RowReader` object. Making the name `createFileReader` would cause confusion so I think we should still use the name `createReader()`. 
I will comment the functions.